package/Dockerfile

```diff
@@ -1,3 +1,7 @@
-FROM alpine
+FROM registry.suse.com/bci/bci-base:15.3
+RUN zypper -n update && \
+    zypper -n clean -a && \
+    rm -rf /tmp/* /var/tmp/* /usr/share/doc/packages/* /usr/share/doc/manual/* /var/log/*
 COPY bin/webhook /usr/bin/
+USER 1000
```

Comment:
BCI base does not appear to have a user with an ID of 1000 by default.[Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user), indicate that you are going to want to create the user before attempting to use it. I'm unsure of the exact impact of trying to use a user like this which doesn't have a corresponding entry in /etc/passwd, but I would prefer that we add the user first unless you have a specific reason not to.